eureka-client/src/main/java/com/netflix/discovery/shared/transport/jersey/JerseyEurekaHttpClientFactory.java

```diff
@@ -112,13 +116,18 @@ public void shutdown() {
     public static JerseyEurekaHttpClientFactory create(EurekaClientConfig clientConfig,
                                                        Collection<ClientFilter> additionalFilters,
                                                        InstanceInfo myInstanceInfo,
-                                                       AbstractEurekaIdentity clientIdentity) {
```

Comment:
I know this is very unlikely to be used outside of internal classes, but give that the method has a public signature, do you mind creating a new overload with the optionals and delegate over (rather than changing) from the original? Thanks.